src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -2257,6 +2441,7 @@ private int calculateSubscribeFlags(
         private final class Subscription
         {
             private int id = 0;
+            private int options = 0;
```

Comment:
Make sure to avoid explicitly initializing non-final member fields to built-in Java defaults, such as `0` for numbers and `null` for objects.